src/main/java/core/levels/LevelManager.java

```diff
@@ -28,6 +28,10 @@
     private final OrthogonalTiledMapRenderer mapRenderer;
     private final WorldEntityManager entityManager;
     private final SpriteBatch batch;
+    static final int SPAWN_BUFFER = 15;
+    static final int VELOCITY_ITERATIONS = 6;
+    static final int POSITION_ITERATIONS = 2;
```

Comment:
thsese can be inlined with an added comment, they are default values suggested for box2d